03_Modules/SmartCharging/src/module/api.ts

```diff
@@ -59,12 +60,30 @@ export class SmartChargingModuleApi
     CallAction.ClearChargingProfile,
     ClearChargingProfileRequestSchema,
   )
-  clearChargingProfile(
+  async clearChargingProfile(
     identifier: string,
     tenantId: string,
     request: ClearChargingProfileRequest,
     callbackUrl?: string,
   ): Promise<IMessageConfirmation> {
+    if (!request.chargingProfileId && !request.chargingProfileCriteria) {
```

Comment:
What about the case `include one or more of the fields stackLevel, evseId and chargingProfilePurpose` at `K10.FR.02` Would that already be covered here?

Reply:
Thanks a lot for the helpful review, @ChrisWeissmann ! I fixed the code based on your feedback.